aether/sdk/tests/test_commands.py

```diff
@@ -281,6 +281,13 @@ def test_cdn_publish(self, mock_cdn_save):
                      stdout=self.out,
                      stderr=self.out)
 
+        call_command('cdn_publish',
+                     '--cdn-url=http://cdn-server/path/to/',
+                     '-w=aether/sdk/tests/webpackfiles',
+                     '--storage-path=/__assets__/sdk/',
+                     stdout=self.out,
+                     stderr=self.out)
```

Comment:
duplicated?

Reply:
to cover the if conditions for when cdn_url and storage_path end and start with `/` respectively

Reply:
in that case I would split it in two tests with different values, you'll never know if the following check belongs to the first or the second call :woman_shrugging: